pkg/volume/configmap/configmap.go

```diff
@@ -202,22 +202,36 @@ func (b *configMapVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	return nil
 }
 
-func makePayload(mappings []api.KeyToPath, configMap *api.ConfigMap) (map[string][]byte, error) {
-	payload := make(map[string][]byte, len(configMap.Data))
+func makePayload(mappings []api.KeyToPath, configMap *api.ConfigMap, defaultMode *int32) (map[string]volumeutil.FileProjection, error) {
+	if defaultMode == nil {
+		return nil, fmt.Errorf("No defaultMode used, not even the default value for it")
```

Comment:
And `%v` is nil. Can I change this in another PR, so this goes in with all the changes to generated code, etc.?

In any case, this has a default value and should not happen normally, just when created manually or so (when default don't apply) and is just to avoid the null pointer deference.